### PR TITLE
Stop preloading imagery tiles except whenanimating

### PR DESCRIPTION
**Pull request by tephenavies**
### What this PR does

Only preload the next timestep for a WMS & ArcGIS MapServer timeseries item when animating the item.

Waiting on testing from GA to merge.

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ ] I've updated relevant documentation in `doc/`.
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Review by nf-s — APPROVED** [review]
I approve this change - if you add to `CHANGES.md`

**Comment by staffordsmith83** [comment]
Hi @steve9164 and @nf-s whats the status of this PR? Are we still "Waiting on testing from GA to merge" ?

**Comment by tephenavies** [comment]
I think we just merge this. No need to wait for externals.

**Review by nf-s — APPROVED** [review]

**Pull request merged**
